app/reflexes/bulk_actions_in_orders_list_reflex.rb

```diff
@@ -2,22 +2,29 @@
 
 class BulkActionsInOrdersListReflex < ApplicationReflex
   def resend_confirmation_email(order_ids)
-    Spree::Order.where(id: order_ids).find_each do |o|
+    orders(order_ids).find_each do |o|
       Spree::OrderMailer.confirm_email_for_customer(o.id, true).deliver_later if can? :resend, o
     end
 
-    flash[:success] = I18n.t("admin.resend_confirmation_emails_feedback", count: order_ids.count)
-    cable_ready.dispatch_event(name: "modal:close")
-    morph "#flashes", render(partial: "shared/flashes", locals: { flashes: flash })
+    success("admin.resend_confirmation_emails_feedback", order_ids.count)
   end
 
   def send_invoice(order_ids)
-    Spree::Order.where(id: order_ids).find_each do |o|
+    orders(order_ids).find_each do |o|
       Spree::OrderMailer.invoice_email(o.id).deliver_later unless o.distributor.can_invoice?
     end
 
-    flash[:success] = I18n.t("admin.send_invoice_feedback", count: order_ids.count)
+    success("admin.send_invoice_feedback", order_ids.count)
+  end
+
+  # private
```

Comment:
Can this be un-commented?
```suggestion
  private
```

Reply:
Sure, thanks!